.expeditor/verify_private.pipeline.yml

```diff
@@ -119,6 +119,7 @@ steps:
     - export SEMGREP_JOB_URL=$BUILDKITE_BUILD_URL
     - export SEMGREP_BRANCH=$BUILDKITE_BRANCH
     - export SEMGREP_REPO_NAME=chef/automate
+    - export SEMGREP_REPO_URL=https://github.com/chef/automate
```

Comment:
This is the secret sauce for the GitHub commits.

Reply:
A code comment couldn't hurt here. If I were to change this in a way that might break the functionality (or just look at this repo as an example semgrep configuration), I might not know what to test without a hint.